tests/functional/object/mcg/test_noobaa_db_tls.py

```
@@ -0,0 +1,202 @@
+        # ------------------------------------------------------------------
+        # Step 3: Verify the NooBaa core pod is configured to require PostgreSQL
+        # TLS. The noobaa-operator sets POSTGRES_SSL_REQUIRED=true (and
+        # POSTGRES_SSL_UNAUTHORIZED=true) when CNPG TLS is enabled
+        # ------------------------------------------------------------------
+        core_env = self._get_noobaa_container_envs()[constants.NOOBAA_CORE_STATEFULSET]
+        core_env_map = {env["name"]: env.get("value") for env in core_env}
+        ssl_required = core_env_map.get(constants.POSTGRES_SSL_REQUIRED_ENV)
+        logger.info(
+            f"NooBaa core {constants.POSTGRES_SSL_REQUIRED_ENV}={ssl_required}, "
+            f"{constants.POSTGRES_SSL_UNAUTHORIZED_ENV}="
+            f"{core_env_map.get(constants.POSTGRES_SSL_UNAUTHORIZED_ENV)}"
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)